MyPerfectOnboardingApplication/MyPerfectOnboarding.Api.Tests/Controllers/ListControllerTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using MyPerfectOnboarding.Api.Controllers;
+using MyPerfectOnboarding.Api.Models;
+using MyPerfectOnboarding.Api.Tests.Utils;
+using NUnit.Framework;
+
+namespace MyPerfectOnboarding.Api.Tests.Controllers
+{
+    [TestFixture]
+    class ListControllerTests
+    {
+        private ListController _listController;
+
+        [SetUp]
+        public void Init()
+        {
+            _listController = new ListController
+            {
+                Request = new HttpRequestMessage(),
+                Configuration = new HttpConfiguration()
+            };
+        }
+
+        [Test]
+        public async Task Get_ReturnsListOfItems()
+        {
+            var response = await _listController.GetAsync();
+
+            var message = await response.ExecuteAction();
+
+            List<ListItem> items;
+
+            message.TryGetContentValue(out items);
+
+            Assert.That(message.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+
+            Assert.That(items, Is.InstanceOf<List<ListItem>>());
+        }
+
+        [Test]
+        public async Task Get_ReturnsItemWithGivenId()
+        {
+            Guid id = _listController.items.First().Id;
+
+            var response = await _listController.GetAsync(id);
```

Comment:
Again, it is a dummy test. Send just plain Id string like you have in the items ("aaaa").